frontend/src/content/news.json

```
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-30",
+      "title": "Metabolic Atlas 3.0",
```

I'd like to also mention GotEnzymes in the title:
```suggestion
      "title": "Metabolic Atlas 3.0 presents GotEnzymes",
```